examples/discovery_example2.py

```diff
@@ -0,0 +1,25 @@
+"""
+Example that shows how to list available chromecasts.
+"""
+import argparse
+import logging
+import time
+
+import pychromecast
+import zeroconf
+
+parser = argparse.ArgumentParser(description="Example on how to receive updates on discovered chromecasts.")
+parser.add_argument("--show-debug", help="Enable debug log", action="store_true")
+args = parser.parse_args()
+
+if args.show_debug:
+    logging.basicConfig(level=logging.DEBUG)
+
+devices, browser = pychromecast.discovery.discover_chromecasts()
+# Shut down discovery
+pychromecast.stop_discovery(browser)
```

Comment:
I am getting this:
AttributeError: module 'pychromecast' has no attribute 'stop_discovery'

Reply:
Did you include all the changes from the PR?
`stop_discovery` is readded to pychromecast here: https://github.com/home-assistant-libs/pychromecast/blob/b43127e0e6d884356ef3622d72a468e2eb544346/pychromecast/__init__.py#L21

Reply:
oops.. you're right :P